css/_variables.scss

```
@@ -38,3 +38,12 @@ $header-bg-color: #fff;
 // left sidebar width + right sidebar width
 $sidebarsWidthDesktop: 655px;
 $sidebarsWidthTablet: 594px;
+
+$transition-duration: 0.3s;
+$transition-function: linear;
```

To avoid duplication.
```suggestion
$transition-duration: 0.3s;
$transition-function: linear;
$transition-base: $transition-duration $transition-function;
```